e2e/scenario/validators.go

```diff
@@ -47,6 +48,34 @@ func SysctlConfigValidator(customSysctls map[string]string) *LiveVMValidator {
 	}
 }
 
+func KubeletConfigValidator(kubeletConfigParams map[string]string) *LiveVMValidator {
+	keysToCheck := make([]string, 0)
+	for k, _ := range kubeletConfigParams {
+		keysToCheck = append(keysToCheck, k)
```

Comment:
I don't think you want this.

Reply:
Hmm, its so I can generate that json file in stdout with jq